cli/src/context.rs

```diff
@@ -29,7 +29,14 @@ pub struct ConfigFile {
 
 fn read_or_default<T: DeserializeOwned + Default>(path: PathBuf) -> Result<T> {
     match std::fs::read_to_string(path) {
-        Ok(contents) => Ok(toml::from_str(&contents)?),
+        Ok(contents) => {
+            // Empty `credentials.toml` files cannot be parsed.
+            if contents.trim().is_empty() {
+                Ok(T::default())
+            } else {
+                Ok(toml::from_str(&contents)?)
+            }
+        }
```

Comment:
I'd be more inclined to do this:

```rust
pub struct CredentialsFile {
    #[serde(default)]
    pub profile: BTreeMap<String, ProfileCredentials>,
}
```

What do you think?

Reply:
That's much cleaner, thanks! Added in 5f83cc0